core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -2178,6 +2178,18 @@ protected function buildManagerImagePreview($path, $ext, $width, $height, $bases
             } catch (Exception $E) {
                 $timestamp = 0;
             }
+
+            if ($bases['pathIsRelative']) {
+                $absolute_path = $bases['pathAbsolute'] . $path;
+            }
+            $exif = exif_read_data($absolute_path);
```

Comment:
This requires the Exif extension to be enabled.

Reply:
if `$exif` will be empty, nothing will broke

Reply:
Does the function `exif_read_data` exist when the Exif extension is disabled?

Reply:
Is this will be enough? `$exif = @exif_read_data($absolute_path);` Or should I check `function_exists`?